spec/models/post_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'rails_helper'
 
+# disabling tests as this is legacy code which we are not using
```

Comment:
is there any unit tests to test when a user is logged in?

Reply:
Took out that line as it was wrong! Don't think we need unit tests for Post as we haven't added any methods beyond in-built active record etc. Let's get the checker working and we may have to reassess!